src/elements/ts-settings/ts-settings.html

```diff
@@ -296,6 +296,7 @@
             if (name === "serversuite") {
                 App.configurator.setUserSetting("dataserver", value.dataserver);
                 App.configurator.setUserSetting("mediaserver", value.mediaserver);
+                App.configurator.setUserSetting("readerserver", value.mediaserver);
```

Comment:
This should be `value.readerserver`, not `value.mediaserver`.

Reply:
great catch! I pushed the correct one.